src/components/SearchBar.tsx

```diff
@@ -12,9 +12,11 @@ import SearchBarNativeComponent, {
 } from '../fabric/SearchBarNativeComponent';
 
 export const NativeSearchBar: React.ComponentType<SearchBarProps> &
-  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+  typeof NativeSearchBarCommands =
+  SearchBarNativeComponent as unknown as React.ComponentType<SearchBarProps> &
+    SearchBarCommandsType;
 export const NativeSearchBarCommands: SearchBarCommandsType =
-  SearchBarNativeCommands as any;
+  SearchBarNativeCommands as SearchBarCommandsType;
```

Comment:
These types do scare me, I guess it works, but we need to cleanup this mess. Let's proceed now, we will do it in separate iteration on this file.